workshop/modules/terraform-aws-eks-data-addons/variables.tf

```
@@ -125,3 +125,16 @@ variable "emr_spark_operator_helm_config" {
   type        = any
   default     = {}
 }
+
+#-----------JupyterHub-------------
+variable "enable_jupyterhub" {
+  description = "Enable Jupyterhub Add-On"
+  type        = bool
+  default     = true
```

I think we want this off by default
```suggestion
  default     = false
```